Dockerfile.arm

```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1
```

Comment:
Would there be a way to make the arm file depend on the x64 file? 🤔  Let me know, I dislike to just have two things to maintain..